.github/workflows/generate-chart-readme.yml

```diff
@@ -0,0 +1,57 @@
+name: Generate Chart Readme
```

Comment:
Step number one will be done by the tool itself, there is no need to do it twice from my point. The job in charge of update the README will fail in case the metadata is wrong.
About step two, I would discard the second option since it would require new PRs, from my point the README should be updated in the same PR that the metadata is added. Pushing the changes to the users' forks should be enough (we are already requesting to allow edition mantainers). Also, about the trigger, do we actually need a trigger? Couldn't it be run just after creating the PR? In that case, the users will know immediately if the metadata is wrong.

Reply:
Yes, I agree that’s the most straightforward option but I have two concerns:
- I think a bot firing commits is not a good developer experience. In a simple case where I submit a PR that is perfectly prepared then I get the commit of the bot and it is ready to merge, but if I submit a PR with a typo in a parameter description (I mean something like writing “conteiner”, not a syntax error in the metadata) the bot will commit the README, I will have to pull the branch and push the changes. Then a reviewer may make me remove a parameter: pull and push again.
- It mixes two tasks, lint and publish. With two workflows we get the syntax check (we can also verify what is going to be the output in the Action logs) and then once we are sure everything is ok we trigger the workflow to commit the README modification via comment/label/etc.

About doing it in the same PR I agree 100%, since right now documenting the additions in the README is a requirement to accept the changes.

Reply:
I understand the concern, maybe we can catch an `/lgtm` to commit the README. I guess it would be the same job but one that commits changes and the other that doesn't commit the changes?

In any case, I think executing a `git pull` should be pretty straight for the users.